src/test/java/spoon/getExecutableDeclarationBugTest/testClasses/ChangePasswordValidator.java

```diff
@@ -0,0 +1,43 @@
+package pt.ist.socialsoftware.edition.ldod.validator;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.security.crypto.password.PasswordEncoder;
```

Comment:
The spoon project doesn't currently have any dependency on the Spring Framework so these imports/class dependencies will always cause the test to fail.